.github/actions/setup_env/action.yml

```diff
@@ -65,6 +65,11 @@ runs:
         build/**/log.txt
         build/**/*.log
 
+  - name: yosys
+    shell: bash
+    run: |
+      make install_yosys
```

Comment:
The only CI check I'm aware of that was related to this is the `xilinx_yosys_wafove.yaml` test. However, it turns out that that specific flow doesn't use Yosys at all. I don't know who created it or named it. So as far as I'm aware, this is the first time since I've been on the project that we have been installing Yosys using our make target, and unit testing it as part of a flow.